lib/util/keywords.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
```

Comment:
If you could copy the names from `dot-notation` and add your name to the list, it should work.